### PR TITLE
cb integration

### PR DESCRIPTION
**Pull request by yellowHatpro**
- addition of 2 new activities,
    -  the first one is the list of reviews on the basis of entities
    - another activity is the description that shows ratings and text
- remaining:
    - CB API still has some problems so the limit and offset have to change to get the correct results
    - critic id to critic name has to be done

### CODE REVIEW

File: app/src/main/java/org/metabrainz/android/presentation/features/critiques/CritiqueActivity.kt [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                        colorResource(
+                            id = R.color.mb_purple_medium
+                        )
+                    else colorResource(id = R.color.light_gray)
+                )
+                .padding(12.dp)) {
+                Row {
+                    Text(
+                        text = entities[it].displayName,
+                        color = Color.Black,
```
Comment (akshaaatt) [inline app/src/main/java/org/metabrainz/android/presentation/features/critiques/CritiqueActivity.kt:185]: I think white color would be a better preference here.

**Review by akshaaatt — CHANGES_REQUESTED** [review]
Thank you for working on this! 💯 
Just a few changes and then we should be good to merge.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -24,7 +24,12 @@
         android:theme="@style/AppTheme"
         android:usesCleartextTraffic="true"
         tools:targetApi="32">
-
+        <activity
+            android:name=".presentation.features.critiques.ReviewDescriptionActivity"
+            android:exported="false" />
```
[the remark this answers is not in the document]
Comment (yellowHatpro) [reply app/src/main/AndroidManifest.xml]: removed exported tag

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -24,7 +24,12 @@
...
+        <activity
+            android:name=".presentation.features.critiques.CritiqueActivity"
+            android:exported="false" />
```
[the remark this answers is not in the document]
Comment (yellowHatpro) [reply app/src/main/AndroidManifest.xml]: removed exported tag

File: app/src/main/java/org/metabrainz/android/presentation/components/BottomNavigationBar.kt [hunk 4]
```diff
@@ -59,7 +59,10 @@ fun BottomNavigationBar(activity: Activity) {
                             }
                         }
                         "critiques" -> {
-                            FinestWebView.Builder(activity).show("https://critiquebrainz.org/")
```
[the remark this answers is not in the document]
Comment (yellowHatpro) [reply app/src/main/java/org/metabrainz/android/presentation/components/BottomNavigationBar.kt:62]: removed web view dependency completely

[the remark this answers is not in the document]
Comment (yellowHatpro) [reply app/src/main/java/org/metabrainz/android/presentation/components/BottomNavigationBar.kt:62]: removed web view dependency completely

File: app/src/main/java/org/metabrainz/android/presentation/features/critiques/CritiqueActivity.kt (hunk 1 above)
Comment (yellowHatpro) [reply app/src/main/java/org/metabrainz/android/presentation/features/critiques/CritiqueActivity.kt]: color changed to white

**Comment by akshaaatt** [comment]
Retiring the work here. Please have a look at https://blog.metabrainz.org/2022/10/13/mobile-apps-lets-welcome-the-listenbrainz-app/ and look to add this to https://github.com/metabrainz/listenbrainz-android

**Pull request closed without merging**